src/svm.py

```diff
@@ -0,0 +1,84 @@
+"""
+file of Support Vector Machines
+"""
+import os
+from datetime import datetime
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
+from sklearn import svm
+from sklearn.metrics import plot_confusion_matrix
+from sklearn.model_selection import GridSearchCV
+
+
+class SupportVectorMachines:  # pylint: disable= R0902
+    """
+    class  Support Vector Machines  classification
+    """
+
+    def __str__(self):
+        return self.__class__.__name__
+
+    def __init__(self, x_data, y_data):
+        """
+            constructor with parameters   of  Support Vector Machines class
+        """
+        self.x_data = x_data
+        self.y_data = y_data
+        self.clf = None
+        self.x_train, self.x_test, self.y_train, self.y_test = train_test_split(x_data, y_data, test_size=0.2)
+        self.svc_model = svm.SVC(kernel='linear')
+
+    def s_v_m(self):
+        """
+             function svm for  predicting score and gird search
+        """
+
+        self.svc_model.fit(self.x_train, self.y_train)
+        # data = pd.DataFrame({'Actual': self.y_test, 'Predicted': y_pred})
+        score = self.svc_model.score(self.x_train, self.y_train)
+        print(score)
```

Comment:
You can delete these lines because you want to run fit with all sets of parameters.
```
- self.svc_model.fit(self.x_train, self.y_train)
- #data = pd.DataFrame({'Actual': self.y_test, 'Predicted': y_pred})
- score = self.svc_model.score(self.x_train, self.y_train)
- print(score)
```